### PR TITLE
[PR-REF 225-229]-[IMP]-Improved demo data file and added group in dl total revenue

### PR DESCRIPTION
**Pull request by YogeshMahera-SerpentCS**
Fixed Issue of this comments #225 #229 

>  If the user is not in estimation group he should not see the total of deliverable in the smart button for deliverables
> Demo product has no cost set. SER01 should be 25, SER02->50, SER03->30, WEB01->100, WEB02->3639.

### CODE REVIEW

**Comment by YogeshMahera-SerpentCS** [comment]
@elicoidal 
Please review

**Comment by elicoidal** [comment]
@YogeshMahera-SerpentCS the smart button display and product Costs are OK.
Some other small details:
- [x] the currency in tree view should not be displayed if the user not in estimation group.
- [x] in deliverable tree view, uom field should be after the quantity

File: business_requirement/views/business_view.xml [hunk 1]
```diff
@@ -9,6 +9,13 @@
             'search_default_parent_id': [active_id],
             'default_parent_id': active_id,
         }</field>
+        <field name="help" type="html">
```
Comment (elicoidal) [inline business_requirement/views/business_view.xml]: I think this is not correct. There is already a standard help description + this text is for project, not BR

**Review by elicoidal — CHANGES_REQUESTED** [review]
almost there

File: business_requirement/views/business_view.xml (hunk 1 above)
Comment (elicoidal) [reply business_requirement/views/business_view.xml]: In short: keep the original helper message

Comment (YogeshMahera-SerpentCS) [reply business_requirement/views/business_view.xml]: Yes, got it, if project then it should be over here
https://github.com/OCA/business-requirement/blob/10.0/business_requirement/views/project.xml#L5
right?
@elicoidal

Comment (elicoidal) [reply business_requirement/views/business_view.xml]: No: I mean we should not touch the helper message for project and the one for BR is already fixed in the original module (so normally there is nothing to do in this module and you should remove this piece of code).
Let me know if unclear

**Comment by YogeshMahera-SerpentCS** [comment]
@elicoidal
#249 Fixed this issue in this PR
Added groups in selection. AMENDED changes 

peek 2017-09-18 15-48

**Comment by YogeshMahera-SerpentCS** [comment]
Fixed issue of #229  
> One last detail: in deliverable form view, the Currency field should be after the total revenue field.

Fixed in this PR.

**Comment by elicoidal** [comment]
@YogeshMahera-SerpentCS Thanks: it seems that the group cost is not included in your changes (should inherit from estimation AFAIK). Can you recheck?
I will test it as soon as runbot is ready

**Comment by YogeshMahera-SerpentCS** [comment]
@elicoidal Yes right.I'll do needful changes.
Thanks

**Comment by elicoidal** [comment]
Besides, can you put both groups together if possible?

**Comment by YogeshMahera-SerpentCS** [comment]
You mean that 
Business Requirement Cost --> Estimation group and Cost control group together in selection?

**Comment by elicoidal** [comment]
Yes: cost group should depend on estimation group

**Comment by elicoidal** [comment]
> Besides, can you put both groups together if possible?

I mean in the list above, the 2 selection boxes are separated by the group "employee". they should be together

**Comment by YogeshMahera-SerpentCS** [comment]
peek 2017-09-18 17-02grop

Is this the way you want?

**Comment by elicoidal** [comment]
:+1:
 




------------------
--
 	    Eric Caudal [Founder and CEO]
                              Skype: elico.corp. Phone: + 86 186 2136 1670 (Cell), + 86 21 6211 8017/27/37 (Office)             
             Elico Shanghai (Hong Kong/Shenzhen/Singapore)             
Odoo Gold Partner // Best Odoo Partner APAC 2014 and 2016
 

 
 
 
------------------ Original ------------------
From: "Yogesh Mahera"; 
Date: 2017年9月18日(星期一) 晚上7:34
To: "OCA/business-requirement"; 
Cc: "Eric @ Elico Corp"; "Mention"; 
Subject: Re: [OCA/business-requirement] [PR-REF 225-229]-[IMP]-Improved demodata file and added group in dl total revenue (#248)

 


 
Is this the way you want?

**Comment by YogeshMahera-SerpentCS** [comment]
@elicoidal 
Cost control group as selection field you can test here #225

**Review by elicoidal — APPROVED** [review]

**Comment by elicoidal** [comment]
@victormartinelicocorp

**Review by martintorresvm — APPROVED** [review]

**Comment by elicoidal** [comment]
@YogeshMahera-SerpentCS Can you rebase this one so that we can merge it?

**Comment by YogeshMahera-SerpentCS** [comment]
@elicoidal rebased

**Pull request merged**